docs/release-notes/breaking-changes.md

```diff
@@ -0,0 +1,53 @@
+---
+navigation_title: "Elastic APM"
+---
+
+# Elastic APM breaking changes
+
+Before you upgrade, carefully review the Elastic APM breaking changes and take the necessary steps to mitigate any issues.
+
+% To learn how to upgrade, check out <uprade docs>.
+
+% ## Next version
+
+% **Release date:** Month day, year
+
+% ::::{dropdown} Title of breaking change
+% Description of the breaking change.
+% For more information, check [PR #](PR link).
+% **Impact**<br> Impact of the breaking change.
+% **Action**<br> Steps for mitigating deprecation impact.
+% ::::
+
+## 9.0.0 [9-0-0]
+
+**Release date:** March 25, 2025
+
+::::{dropdown} Change server information endpoint "/" to only accept GET and HEAD requests
```

Comment:
thanks for bringing this in from https://github.com/elastic/docs-content/pull/647 already!